### DISCUSSION TITLE
UPS for Raspberry Pi 3A

### DISCUSSION THREAD

**Discussion by spitfire** [jayofelony/pwnagotchi#9] [Q&A]
I’m looking for a UPS hat for Raspberry Pi 3A that is supported by some UPS plugin for pwnagotchi, will [this one](https://www.dfrobot.com/product-1825.html) work?

**Comment by kagehisa** [comment]
It will propably work on the pi, but I don't know if there is a pwnagotchi plugin for it.

**Reply by spitfire** [reply]
That's generally what I'm asking for. I've seen your other thread, and I'd like to find something that works and fits RPi3A well, with reasonably big battery too - so nothing tiny like the [UPS Lite](https://www.amazon.pl/gp/product/B09G35NLMW) or huge like [Waveshare UPS HAT](https://www.waveshare.com/wiki/UPS_HAT) (which is great on my RPi4, but too big for RPi3A.

**Reply by kagehisa** [reply]
The only other options that comes to mind and should work with the pisugar2 pwnagotchi plugin would be pisugar2-plus. But take this info with a grain of salt, I have not tested it with the pwnagotchi plugin since I don't use  this unit.

**Reply by spitfire** [reply]
Yeah, I've seen it (and the plugins for it) but it's more than twice as expensive as the other ones I was looking at. It does look like a good fit and has a huge battery, so if I can't find something reasonably inexpensive I might think about it.

**Reply by spitfire** [reply]
I think I'll settle on the Pisugar 3 plus, just need to confirm that it 100% fits Pi3A (physically, as in PCB not being larger than Pi's), GPIO scheme is the same on all Pis, right?
